geth/account/interfaces.go

```diff
@@ -0,0 +1,45 @@
+package account
+
+import (
+	"crypto/ecdsa"
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/status-im/status-go/extkeys"
+)
+
+type accountNode interface {
```

Comment:
I don't clearly see what exactly `accountNode` means. It sounds like `node with account` or `account-enabled node`, but I suppose it's neither of both.

Reply:
This is interface, which show what node manager methods are really used by account manager. I think you could suggest name for this interface.